### PR TITLE
Enum filter driver

### PR DESCRIPTION
**Pull request by kapn-kaos**
#### Synopsis
List the installed filter drivers

#### Methodology
* Opens the service registry key
* Iterates through each service
* Attempts to open Instances subkey
* Queries Altitude value
* Compares Altitude value to determine filter driver type

#### Logs
- [enum_filter_driver.cppcheck.log](https://github.com/trustedsec/CS-Situational-Awareness-BOF/files/6356531/enum_filter_driver.cppcheck.log)
- [enum_filter_driver.Win7.x64.command.log](https://github.com/trustedsec/CS-Situational-Awareness-BOF/files/6356532/enum_filter_driver.Win7.x64.command.log)
- [enum_filter_driver.Win7.x64.drmemory.log](https://github.com/trustedsec/CS-Situational-Awareness-BOF/files/6356533/enum_filter_driver.Win7.x64.drmemory.log)
- [enum_filter_driver.Win7.x64.standalone.log](https://github.com/trustedsec/CS-Situational-Awareness-BOF/files/6356534/enum_filter_driver.Win7.x64.standalone.log)
- [enum_filter_driver.Win7.x86.command.log](https://github.com/trustedsec/CS-Situational-Awareness-BOF/files/6356535/enum_filter_driver.Win7.x86.command.log)
- [enum_filter_driver.Win7.x86.drmemory.log](https://github.com/trustedsec/CS-Situational-Awareness-BOF/files/6356536/enum_filter_driver.Win7.x86.drmemory.log)
- [enum_filter_driver.Win7.x86.standalone.log](https://github.com/trustedsec/CS-Situational-Awareness-BOF/files/6356537/enum_filter_driver.Win7.x86.standalone.log)
- [enum_filter_driver.Win10.x64.command.log](https://github.com/trustedsec/CS-Situational-Awareness-BOF/files/6356538/enum_filter_driver.Win10.x64.command.log)
- [enum_filter_driver.Win10.x64.standalone.log](https://github.com/trustedsec/CS-Situational-Awareness-BOF/files/6356539/enum_filter_driver.Win10.x64.standalone.log)
- [enum_filter_driver.Win10.x86.command.log](https://github.com/trustedsec/CS-Situational-Awareness-BOF/files/6356540/enum_filter_driver.Win10.x86.command.log)
- [enum_filter_driver.Win10.x86.standalone.log](https://github.com/trustedsec/CS-Situational-Awareness-BOF/files/6356541/enum_filter_driver.Win10.x86.standalone.log)

### CODE REVIEW

File: SA/SA.cna [hunk 1]
```diff
@@ -1060,3 +1060,38 @@ beacon_command_register(
...
+
+	if ((size(@_) < 1) || (size(@_) > 2))
+	{
+		berror($1, beacon_command_detail("enum_filter_driver"));
+		berror($1, "Invalid number of arguments");
+		return;
+	}
+	$system = iff(-istrue $2, $2, $null);
+	blog($1, "Retrieving list of filter drivers");
+	$args = bof_pack($1, "Z", $system);
```
Comment (freefirex) [inline SA/SA.cna:1078]: This needs to be a lowercase z since multibyte is used instead of unicode funcs.

**Review by freefirex — CHANGES_REQUESTED** [review]
Just the one change needed to enable the remote codepath to work as intended, otherwise looks good!

File: SA/SA.cna (hunk 1 above)
Comment (kapn-kaos) [reply SA/SA.cna]: Updated in the most recent commit

**Review by freefirex — APPROVED** [review]
Looks good 💯

**Pull request merged**